### PR TITLE
Updated strings for LABY.net

### PR DESCRIPTION
**Pull request by Northernside**
Heya!
I updated some strings for the LABY.net Presence. Proof it's working:
https://imgur.com/a/x6c3SsX

### CODE REVIEW

File: websites/L/LABY.net/presence.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  presenceData.startTimestamp = browsingStamp;
+  if(document.location.pathname.startsWith("/skins")) 
+    presenceData.details = "Viewing skins";
+  else if(document.location.pathname.startsWith("/skin")) 
+    presenceData.details = "Viewing skins";
+  else if(document.location.pathname.startsWith("/cloaks")) 
+    presenceData.details = "Viewing LabyMod cloaks";
+  else if(document.location.pathname.startsWith("/capes")) 
+    presenceData.details = "Viewing Minecraft capes";
+  else if(document.location.pathname.startsWith("/@"))
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:21]:
```suggestion
  else if(document.location.pathname.startsWith("/@")) {
    presenceData.details = "Viewing profile";
    presenceData.state = document.location.pathname.split("/@")[1]
  } else if(document.location.pathname.startsWith("/settings")) 
```

File: websites/L/LABY.net/presence.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    presenceData.details = "Viewing LabyMod cloaks";
+  else if(document.location.pathname.startsWith("/capes")) 
+    presenceData.details = "Viewing Minecraft capes";
+  else if(document.location.pathname.startsWith("/@")) 
+    presenceData.details = `Viewing ${document.location.pathname.split("/@")[1]}'s profile`;
+  else if(document.location.pathname.startsWith("/settings")) 
+    presenceData.details = "Viewing profile settings";
+  else if(document.location.pathname.startsWith("/server")) {
+    const item = document.querySelector("div.server-info-wrapper>h1").textContent;
+    presenceData.details = `Viewing server: ${item}`;
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:27]:
```suggestion
    presenceData.details = "Viewing server";
    presenceData.state = item;
```

File: websites/L/LABY.net/presence.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  } else if(document.location.pathname.startsWith("/badge")) {
+    const item = document.querySelector("div.mb-1>h1").textContent;
+    presenceData.details = `Viewing badge: ${item}`;
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:30]:
```suggestion
    presenceData.details = "Viewing badge";
    presenceData.state = item;
```

File: websites/L/LABY.net/presence.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  } else if(document.location.pathname.startsWith("/cape")) {
+    const item = document.querySelector("div.mb-1>h1").textContent;
+    presenceData.details = `Viewing cape: ${item}`;
```
Comment (N0chteil) [inline websites/L/LABY.net/presence.ts:33]:
```suggestion
    presenceData.details = "Viewing cape";
    presenceData.state = item;
```

**Review by 7loud — COMMENTED** [review]
Some suggestions

**Review by Slowlife01 — DISMISSED** [review]

File: websites/L/LABY.net/presence.ts [hunk 5]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
     presenceData.details = "Viewing LabyMod cloaks";
   else if(document.location.pathname.startsWith("/capes")) 
     presenceData.details = "Viewing Minecraft capes";
-  else if(document.location.pathname.startsWith("/@")) 
-    presenceData.details = `Viewing ${document.location.pathname.split("/@")[1]}'s profile`;
-  else if(document.location.pathname.startsWith("/settings")) 
+  else if(document.location.pathname.startsWith("/@")) {
+    const item = document.querySelector("div.profile-header>h1").textContent;
+    presenceData.details = "Viewing profile";
+    presenceData.state = item;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:24]:
```suggestion
    const user = document.querySelector("div.profile-header>h1").textContent;
    presenceData.details = "Viewing profile of:";
    presenceData.state = user;
```

File: websites/L/LABY.net/presence.ts [hunk 6]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
+  } else if(document.location.pathname.startsWith("/settings")) 
     presenceData.details = "Viewing profile settings";
   else if(document.location.pathname.startsWith("/server")) {
     const item = document.querySelector("div.server-info-wrapper>h1").textContent;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:28]:
```suggestion
    const server = document.querySelector("div.server-info-wrapper>h1").textContent;
```

File: websites/L/LABY.net/presence.ts [hunk 7]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
-    presenceData.details = `Viewing server ${item}`;
+    presenceData.details = "Viewing server";
+    presenceData.state = item;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:30]:
```suggestion
    presenceData.details = "Viewing server:";
    presenceData.state = server;
```

File: websites/L/LABY.net/presence.ts [hunk 8]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
   } else if(document.location.pathname.startsWith("/badge")) {
     const item = document.querySelector("div.mb-1>h1").textContent;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:32]:
```suggestion
    const badge = document.querySelector("div.mb-1>h1").textContent;
```

File: websites/L/LABY.net/presence.ts [hunk 9]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
-    presenceData.details = `Viewing badge ${item}`;
+    presenceData.details = "Viewing badge";
+    presenceData.state = item;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:34]:
```suggestion
    presenceData.details = "Viewing badge:";
    presenceData.state = badge;
```

File: websites/L/LABY.net/presence.ts [hunk 10]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
   } else if(document.location.pathname.startsWith("/cape")) {
     const item = document.querySelector("div.mb-1>h1").textContent;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:36]:
```suggestion
    const cape = document.querySelector("div.mb-1>h1").textContent;
```

File: websites/L/LABY.net/presence.ts [hunk 11]
```diff
@@ -18,19 +18,24 @@ presence.on("UpdateData", () => {
...
-    presenceData.details = `Viewing cape ${item}`;
+    presenceData.details = "Viewing cape";
+    presenceData.state = item;
```
Comment (Bas950) [inline websites/L/LABY.net/presence.ts:38]:
```suggestion
    presenceData.details = "Viewing cape:";
    presenceData.state = cape;
```

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**